hugegraph-core/src/main/java/com/baidu/hugegraph/StandardHugeGraph.java

```diff
@@ -258,8 +258,13 @@ public void serverStarted(Id serverId, NodeRole serverRole) {
                  serverId, serverRole, this.name);
         this.serverInfoManager().initServerInfo(serverId, serverRole);
 
-        LOG.info("Search olap property key for graph '{}'", this.name);
-        this.schemaTransaction().initAndRegisterOlapTables();
+        // TODO: check necessary?
+        LOG.info("Init olap property-keys for graph '{}'", this.name);
```

Comment:
seems the info is not accurate,  "Check olap property-keys and tables for xxx"  is better